internal/attestations/authorization.go

```diff
@@ -143,7 +143,7 @@ func (a *Attestations) GetReferenceAuthorizationFor(repo *git.Repository, refNam
 // ReferenceAuthorizationPath constructs the expected path on-disk for the
 // reference authorization attestation.
 func ReferenceAuthorizationPath(refName, fromID, toID string) string {
-	return path.Join(refName, fmt.Sprintf("%s-%s", fromID, toID))
+	return filepath.Join(refName, fmt.Sprintf("%s-%s", fromID, toID))
```

Comment:
This path is used within Git to set up subtrees and eventually a blob. So, path.Join is fine.